app/Http/Controllers/Api/V1/Integration/VideoDirectUrlController.php

```diff
@@ -61,6 +61,7 @@ public function getVimeoDirectUrl($url)
     }    
 
     /**
+     * Get Vimeo Video Info
```

Comment:
Add new line between title, description param and return.

Reply:
What about controller methods that are private or used internal without any associated route.
Do they also appear in the generated Docs ?